### PR TITLE
FIX : Missing hook in getElementProperties

### PR DESCRIPTION
**Pull request by atm-irvine**
#FIX|Fix Missing hook in getElementProperties
I added the missing hook in getElementProperties to handle $element_properties array because of some errors for permissions

### CODE REVIEW

File: htdocs/core/lib/functions.lib.php [hunk 1]
```diff
@@ -12154,7 +12166,15 @@ function getElementProperties($element_type)
 		'dir_output' => $dir_output
 	);
 
-	//var_dump($element_properties);
+	$reshook = $hookmanager->executeHooks('changeElementProperties', $parameters);
```
Comment (eldy) [inline htdocs/core/lib/functions.lib.php:12169]: Can youbrename trigger function in getElementProperties to match same name than dolibarr function.

**Review by eldy — CHANGES_REQUESTED** [review]

**Comment by john-botella** [comment]
@eldy this PR allow fix some troubles for modules discribed in [#29369 ](https://github.com/Dolibarr/dolibarr/issues/29369)

File: htdocs/core/lib/functions.lib.php [hunk 2]
```diff
@@ -12154,7 +12166,15 @@ function getElementProperties($element_type)
 		'dir_output' => $dir_output
 	);
 
-	//var_dump($element_properties);
+	$reshook = $hookmanager->executeHooks('getElementProperties', $parameters);
+	if ($reshook < 0) {
+		setEventMessages($hookmanager->error, $hookmanager->errors, 'errors');
```
Comment (eldy) [inline htdocs/core/lib/functions.lib.php:12171]: The method is getElementProperties is low level method that can be run out of user/web context (in cron job for exemple).
So we should not have setEventMessage inside.

**Review by eldy — CHANGES_REQUESTED** [review]

**Comment by john-botella** [comment]
close this PR and use #30086 instead

**Pull request closed without merging**